addons/mrp/models/mrp_workorder.py

```diff
@@ -223,7 +223,7 @@ def _set_qty_producing(self):
         for workorder in self:
             if workorder.qty_producing != 0 and workorder.production_id.qty_producing != workorder.qty_producing:
                 workorder.production_id.qty_producing = workorder.qty_producing
-                workorder.production_id._set_qty_producing()
+                workorder.production_id._set_qty_producing(False)
```

Comment:
This is already done in `17.4`, as far as I can see here:
https://github.com/odoo/odoo/blob/0b0da52a98916c66848573415ac4435c9d4ad010/addons/mrp/models/mrp_workorder.py#L226
Since you already did it in #175816 I guess you can remove it from this PR.